### ISSUE TITLE
Ace Uncaught Typerror

### ISSUE THREAD

**Issue by sqpp15** [ajaxorg/ace#3346]
Hello!

I am using WordPress and this editor but seems there is some issue.

Can you tell me where to start?

```
ace.js:1 Uncaught TypeError: Cannot read property 'ace/ace' of undefined
    at s (ace.js:1)
    at r (ace.js:1)
    at Object.i [as require] (ace.js:1)
    at ace.js:3
    at ace.js:10
```

![screenshot_1](https://user-images.githubusercontent.com/25447110/28248409-a373c54a-6a44-11e7-90ed-2107defa7696.png)

**Comment by nightwing** [comment]
Could you try loading the non-minified version of ace.js to get a more useful error stack.

**Comment by Simran-B** [comment]
FWIW, I got this error too from the WPBackery Page Builder plugin, at least versions 6.6.0 and 6.7.0 (the latter currently being the latest release from 2021-07-06).

The plugin appears to bundle an ancient version of Ace from 2013/2014. I couldn't pinpoint the exact version, but v1.1.0.1, v1.1.1, and v1.1.2 look very similar in code. I replaced the bundled files with newer versions and the error goes away with Ace v1.1.9 or later.

The error appears in the browser console when I load a page for editing, but I can't actually find anything being broken or not working. The plugin seems to be use Ace for a CSS editor, which works fine as far as I can tell.
